{{ cookiecutter.repo_name }}/.pre-commit-config.yaml

```diff
@@ -23,4 +23,4 @@ repos:
   - repo: https://github.com/python-poetry/poetry
     rev: '1.7.0'
     hooks:
-    - id: poetry-lock
+    - id: poetry-check
```

Comment:
Does it not need the `--lock` arg here?

Reply:
`poetry check --lock` refreshes the lock file; `poetry check` just checks compatibility. I think we want the former, what do you think?